wger/exercises/api/views.py

```diff
@@ -206,3 +208,23 @@ class MuscleViewSet(viewsets.ReadOnlyModelViewSet):
     ordering_fields = '__all__'
     filter_fields = ('name',
                      'is_front')
+
+
+class ExerciseInformationViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Exercise.objects.all()
+    serializer_class = ExerciseInformationSerializer
+    ordering_fields = '__all__'
+    filter_fields = ('category',
+                     'creation_date',
+                     'description',
+                     'language',
+                     'muscles',
+                     'muscles_secondary',
+                     'name',
+                     'equipment')
+
+    def exercise_information(request):
```

Comment:
I looked through it and I realized the method was redundant and I have excluded it.